cmd/cli/plugin/login/main.go

```diff
@@ -122,6 +124,7 @@ func login(cmd *cobra.Command, args []string) (err error) {
 		}
 	}
 
+	log.Warning("'tanzu login' is deprecated and shall be removed in a future version. Please use 'tanzu context create' instead.")
```

Comment:
The timing and/or the message content doesn't look quite right when the interactive prompting is involved.

If the user is using interactive mode to add server, it seems weird to show the deprecation notice _after_ all the interaction.
If the user is using interactive mode to select a server, the deprecation message is not quite correct for the situation.
It is tricky to do anything before the prompt since we do not know the user's intention. If we want to, one option is to be general with the message. 
(seeing that you are about to launch into interactive mode...), show:
```
... login deprecated....
  to select endpoint, use ....
  to create server, use ....
```